libfaad/sbr_huff.c

```diff
@@ -108,33 +108,29 @@ static const int8_t f_huffman_env_1_5dB[120][2] = {
 };
 
 static const int8_t t_huffman_env_bal_1_5dB[48][2] = {
-    { -64,   1 },    { -63,   2 },    { -65,   3 },    { -62,   4 },
-    { -66,   5 },    { -61,   6 },    { -67,   7 },    { -60,   8 },
-    { -68,   9 },    {  10,  11 },    { -69, -59 },    {  12,  13 },
-    { -70, -58 },    {  14,  28 },    {  15,  21 },    {  16,  18 },
-    { -57,  17 },    { -71, -56 },    {  19,  20 },    { -88, -87 },
-    { -86, -85 },    {  22,  25 },    {  23,  24 },    { -84, -83 },
-    { -82, -81 },    {  26,  27 },    { -80, -79 },    { -78, -77 },
-    {  29,  36 },    {  30,  33 },    {  31,  32 },    { -76, -75 },
-    { -74, -73 },    {  34,  35 },    { -72, -55 },    { -54, -53 },
-    {  37,  41 },    {  38,  39 },    { -52, -51 },    { -50,  40 },
-    { -49, -48 },    {  42,  45 },    {  43,  44 },    { -47, -46 },
-    { -45, -44 },    {  46,  47 },    { -43, -42 },    { -41, -40 }
+    {-64, 1},     {-62, 2},   {-66, 3},   {-60, 4},     {-68, 5},
```

Comment:
These changes I don't quite understand. Could you elaborate a bit, please?

Reply:
Done.